PlaywrightSolar/PageObjects/SolarShopPage.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightSolar.PageObjects;
+
+public class SolarShopPage(IPage page)
+{
+    private const string SolarShopPageUrl = "https://solartechnology.com.ua/shop";
+   
+    // Methods
+    public async Task VerifyBasketIsEmpty()
+    {
+        await page.WaitForURLAsync(SolarShopPageUrl);
+        
+        var basketIcon = await page.QuerySelectorAsync(".cart-icon");
+        
+        Assert.That(basketIcon, Is.Not.Null, "Basket icon is not found");
+        
+        var basketHasAttribute = await basketIcon!.GetAttributeAsync("class");
+        
+        Assert.That(basketHasAttribute, Does.Contain("cart-icon"), "Basket is not empty");
```

Comment:
you can search by CssSelector and verify in one line like
Assertions.Expect(page.Locator(".cart-icon")).ToBeVisibleAsync()